formats/gallery/Gallery.php

```diff
@@ -360,6 +357,20 @@ private function getImageOverlay() {
 		}
 	}
 
+	/**
+	 * Init standard widget
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return array
```

Comment:
I'd remove everything but the return. The comment suggests a side effect which is not there and the since tag is not helpful for a private function.

Reply:
I like you extracted a function though

Reply:
You are right, of course! I just imitated the other methods... One moment please, I have to clean up the PR!

Reply:
Look: only the return!

Reply:
Please choose sensible clean code over being consistent with existing code, which generally is a mess.

Btw before you invest a lot in testing result formats, you might want to have a look at https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/4313. Without that, or some similar change, it simply is not possible to write good tests.

Reply:
I take you at your word, as this is also my preferred approach. Would you also have changed the other existing comments in this case, just to be consistent? (I would have liked to but preferred to make less noise in the end...)
Thanks for the link, I will have a look at it.

Reply:
Can't fix everything in one go. Lots of issues. Good to focus on high prio ones. Consistency is often overrated.